rfcs/text/0000-zendesk-triage-sop.md

```diff
@@ -0,0 +1,133 @@
+
+**This RFC should not be approved unless funding has been arranged**
+
+### Support Sustainability 
+Upon general availability (GA) of the DCP site to the public, the DCP team expects the rate of issue generation to increase. The current rate of issues is at about 1 per week. A burst of questions is expected upon first contact, which is what was experienced with the Preview site. The rate of tickets will drop to a steady flow after the initial burst, and will slowly increase as the number of active users of the system increases along with the data included and the general utility of the DCP increases.
+Addressing the initial surge of tickets will require a big time commitment. Eventually we will need dedicated staff to handle support. An efficient ticketing triage system and a team well-versed in the DCP will be required to provide a good user experience of the DCP. 
+
+This RFC describes an SOP that should satisfy both the Beta2 test phase and the early days of the product. As we grow in our understanding of user support needs, this SOP will likely need revision to address long-term sustainability.
+
+### **User Stories**
+
+1.  > As a researcher new to the DCP, I would like to get a timely
+    > answer to my question so that I can have a successful experience
+    > submitting, exploring, downloading, or analyzing HCA data.
+
+2.  > As a DCP PM, PO or PL, I would like to be assured that Zendesk
+    > tickets are being responded to in a timely manner so that I can
+    > focus on my own tasks without having to micromanage responses.
+
+3.  > As a DCP team member, I would like to know whether or not I should
+    > be responding to a Zendesk ticket so that I can either respond in
+    > a timely manner or be assured that someone else is responding in a
+    > timely manner.
+
+## Scientific "guardrails" \[optional\]
+
+## Definitions
+`Knowledge Base` - a document that tracks all questions and answers that have been generated through the support infrastructure. This document can be mined to generate FAQs, or finding answers to obscure problems that have been solved in the past.
+`Triager` - the first person to interact with an issue.
+
+## **Detailed Design**
+
+This section describes the flow of an issue through the support state
+diagram presented below.
+
+![](../images/0000-Zendesk-triage-SOP-diagram.jpeg)
+
+## Issue Creation
+
+Any user of the DCP can file an issue. Users can include data contributors, researchers trying to use the data in the DCP, and software developers trying to leverage the DCP API. 
+
+While problems identified by an internal DCP staff member could be filed on Zendesk too, we favor a plan in which those issues are directly added to the applicable development repository, where they can be addressed directly. This plan eliminates the extra work for the triager of managing internally-generated tickets.
```

Comment:
I think this should be more strict. Internal DCP staff MUST NOT file Zendesk issues. The guidance should be crisp as possible. 

_Problems identified by DCP community members MUST not be submitted through the Zendesk process, but filed in the appropriate github repository, where they can be directly addressed by DCP teams._